README.md

```diff
@@ -8,14 +8,18 @@ This project involves creating a responsive website for Pet Care company that fe
 
 ## Setup and Usage
 
-1. Clone the repository to your local machine.
+1. Clone the repository to your local machine with :
+- `git clone https://github.com/Alexandru-Bulai/website-project.git`
+- Open the terminal and go to the folder `cd website-project`
+- Run `npm install` to get all the modules and dependencies ready
+- Open the public and start modify the website at your liking :)
 2. Run `docker-compose up -d` OR `docker compose up -d`.
 3. Visit `http://localhost:8081/home.html`.
 4. You are all set up to edit the webpage! Enjoy 😄
 
 ## Testing enviroment (optional)
 1) In the Test folder you can find the tests runned on JavaScript and you can add more tests there.
-2) Once you finished writing the test simply rum `npm test` on the command line to see if it works.
+2) Once you finished writing the test simply rum `npx jest` or `npx jest /* --watch` on the command line to see if your test passes.
```

Comment:
The commands should be part of the package.json. They should not be different from the ones in the pipeline or anywhere else. Why change it?

Reply:
Right, it should be a uniform experience when running tests, wether is locally or not... i added the `npx jest /* --watch` to run locally without the need to write every time npx jest to do a test

Reply:
No. What I mean was that the instruction should be... run this npm script that takes care of everything. 

Not run npx jest manually. 

My proposal would be to run a script that performs this action:
`docker-compose up -d && jest --testPathPattern=test/* --coverage --watch; docker-compose down`

Add this in your npm scripts. Call it however you like. Could be... called dev. 

Then you can update the readme to say: Type npm run dev and done. You will have docker container opened, and jest running in watch mode. When you close jest, it will remove the container. Everything cleaned.

Reply:
This way you combine the current line 12 with lines 16-18 from the readme into one single script :)